custom/dfci_swasth/tests/test_custom_actions.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime, timedelta
+
+from casexml.apps.case.mock import CaseFactory
+from corehq.apps.data_interfaces.models import AutomaticUpdateRule
+from corehq.apps.data_interfaces.tests.test_auto_case_updates import BaseCaseRuleTest
+from corehq.apps.data_interfaces.tests.util import create_empty_rule
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.form_processor.models import CommCareCase
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+from custom.dfci_swasth.constants import (
+    CASE_TYPE_PATIENT,
+    CASE_TYPE_CASELOAD,
+    PROP_CCUSER_CASELOAD_CASE_ID,
+    PROP_COUNSELLOR_LOAD,
+    PROP_SCREENING_EXP_DATE,
+    PROP_COUNSELLING_EXP_DATE,
+)
+from custom.dfci_swasth.rules.custom_actions import update_counsellor_load
+from dimagi.utils.parsing import ISO_DATE_FORMAT
+
+
+class UpdateCounsellorLoadTest(BaseCaseRuleTest):
+    domain = 'dfci-swasth'
+
+    def setUp(self):
+        super().setUp()
+        self.domain_obj = create_domain(self.domain)
+
+    def tearDown(self):
+        FormProcessorTestUtils.delete_all_cases()
+        super().tearDown()
+
+    def test_non_patient_case(self):
+        patient_rule = create_empty_rule(
```

Comment:
Any reason to create a new one for each test?

Reply:
creating one in setup would also make it easy to delete it in teardown.

Reply:
Updated